pkg/pillar/types/zedroutertypes.go

```diff
@@ -137,13 +137,13 @@ func (status AppNetworkStatus) AwaitingNetwork() bool {
 	return status.AwaitNetworkInstance
 }
 
-// GetULStatusForNI returns UnderlayNetworkStatus for every application VIF
+// GetULStatusForNI returns AppNetAdapterStatus for every application VIF
 // connected to the given network instance (there can be multiple interfaces connected
 // to the same network instance).
-func (status AppNetworkStatus) GetULStatusForNI(netUUID uuid.UUID) []*UnderlayNetworkStatus {
-	var uls []*UnderlayNetworkStatus
-	for i := range status.UnderlayNetworkList {
-		ul := &status.UnderlayNetworkList[i]
+func (status AppNetworkStatus) GetULStatusForNI(netUUID uuid.UUID) []*AppNetAdapterStatus {
+	var uls []*AppNetAdapterStatus
```

Comment:
I think we should also rename `uls` and `ul` variables (there are several occurrences of them across the code base)

Reply:
Perhaps `uls`->`adapters`, `ul` -> `adapter`?

Reply:
I used ana elsewhere but missed this. adapters makes sense here.
Do you think adapter is more clear elsewhere where I'm using ana?

Reply:
I think that I would prefer adapter, but ana is fine too.

Reply:
Yes, this was just copied and can be removed.